_client/src/Views/MainWindow.cpp

```diff
@@ -25,6 +28,51 @@ void MainWindow::setupUI()
     this->showMaximized();
 }
 
+void MainWindow::setupMenu() 
+{
+    // create MenuBar
+    m_mainMenuBar =  menuBar();
+    // File Menu
+    m_fileMenu = createModuleMenu(tr("File"), {ActionTypes::SETTINGS, ActionTypes::EXIT}); 
+    m_mainMenuBar->addMenu(m_fileMenu);
```

Comment:
just `m_mainMenuBar->addMenu(createModuleMenu(tr("File"), {ActionTypes::SETTINGS, ActionTypes::EXIT}))`;

Remove instances of m_fileMenu and others

Reply:
done